File: java/CreateTrunk.java
public class CreateTrunk {
    public static void main(String[] args) throws Exception {
        System.out.println("Creating trunk...");
        
        String accountSid = System.getenv("EXO_ACCOUNT_SID");
        if (accountSid == null) {
            System.err.println("Error: EXO_ACCOUNT_SID is required");
            System.exit(1);
        }
        
        String json = String.format(
            "{\"trunk_name\":\"%s\",\"nso_code\":\"%s\",\"domain_name\":\"%s.pstn.exotel.com\"}",
            _Client.getenvDefault("TRUNK_NAME", "my_ai_trunk"),
            _Client.getenvDefault("NSO_CODE", "ANY-ANY"),
            accountSid
        );
        
        _Client.post("/trunks", json);
        System.out.println("Trunk created successfully!");
    }
} 

File: go/create_trunk.go
package main

import (
    "fmt"
    "os"
)

func main() {
    fmt.Println("Creating trunk...")
    
    trunkData := map[string]string{
        "trunk_name":  getenvDefault("TRUNK_NAME", "my_ai_trunk"),
        "nso_code":    getenvDefault("NSO_CODE", "ANY-ANY"),
        "domain_name": os.Getenv("EXO_ACCOUNT_SID") + ".pstn.exotel.com",
    }
    
    post("/trunks", trunkData)
    fmt.Println("Trunk created successfully!")
} 